Fix accuracy in documentation of Application permissions for azuread_group and azuread_group_member (#1045)
hashicorp/terraform-provider-azuread: docs/resources/group_member.md

The following API permissions are required in order to use this resource.

> When authenticated with a service principal, this resource requires one of the following application roles: `Group.ReadWrite.All` or `Directory.ReadWrite.All`. If the service principal is an owner of the group, these permissions are not required.

Reviewer: ```suggestion
When authenticated with a service principal, this resource requires one of the following application roles: `Group.ReadWrite.All` or `Directory.ReadWrite.All`.

However, if the authenticated service principal is an owner of the group being managed, an application role is not required.
```
